### ISSUE TITLE
Z-Order of Popups and Modals

### ISSUE THREAD

**Issue by leahey** [Megabit/Blazorise#3969]
I have a **DataGrid's** edit popup opening a **Modal** popup, but the Modal appears behind the DataGrid's popup in the z-order. 
Is there a way to specify z-order (like a BringToFront()) or might you have some other suggestion?

**Comment by stsrki** [comment]
Thanks for the report. There is already an open issue for that #3878.

We talked about it irternaly. but we still didn't fix it. The main problem is that users could have as many modals opened at any given time, so the z-order should be dynamically calculated. So we can consider it for one of the next milestones. In the meantime, as a workaround, I can suggest having a CSS selector that changes the z-order of the second modal.

Closing now this ticket since we already have another to track it.
